wordpress-mobile/WordPress-iOS
# Removed the newNavigationBarAppearance feature flag.

Removes the new navigation bar feature flag.

## Suggestion for the code review:

For reviewing the code I suggest starting by reviewing the files with the least changes, and marking them as viewed (GitHub review feature).

Once you're done reviewing a single file, you can tap on the "Viewed" checkbox in the top-right corner:

<img width="157" alt="Screenshot 2021-06-25 at 23 01 11" src="https://user-images.githubusercontent.com/1836005/123484805-407e4980-d609-11eb-9afc-220bbf7c902d.png">

Once you tap it the changes for that file are hidden:

<img width="167" alt="Screenshot 2021-06-25 at 23 02 06" src="https://user-images.githubusercontent.com/1836005/123484893-60ae0880-d609-11eb-9798-9afd6a07684b.png">

## To test:

For testing I recommend that you run the latest release in your device, and run this PR in your simulator to compare the colors.  Also remember to test in both dark and light modes.

- Make sure the quick action buttons coloring looks right.
   - Tap on them and check the background color change when tapped.
   - Make sure the borders look the same.
- Make sure the application navigation bars have the same background and text colors.
- Make sure the navigation bar fonts look the same.
- Make sure the status bar color is the same.
- Go to My Site > Sharing > Facebook > Connect.  Once the web view loads, make sure the title colors look the same.
- Go to My Site > View Site and make sure the title colors look the same.
- Make sure the Me button border and color looks the same.
- Create a new site and accept the quick start help.  When the list of tasks come up, tap on "View your site".  Make sure the popup is dismissed appropiately and the "View Site" option is highlighted under My Site.
- Make sure that the blog detail header view under My Site (the view containing the blog title, url, icon and quick action buttons) looks the same.
- Go to My Site > Switch Site and make sure it looks the same.  Cell rows should not show a disclosure indicator, and the overall look and coloring should be the same as in our previous release.
- Go through the login flow and make sure the navigation bar coloring looks the same.
- Go to the notifications section and make sure it's still showing the large title.
- Make sure the post editor title looks the same.
- Make sure the reader tab still shows a large title and the coloring is right.
- Make sure the Jetpack app main navigation bar color looks right.

## Regression Notes

1. Potential unintended areas of impact

The coloring and styling of navigation bars in the App could be affected if any of the cleanups I did was done incorrectly.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I took several steps to minimize the risks:

   - I tried to avoid making improvements to the code and opted for a very minimalistic cleanup in most cases.  We can always make improvements to the code later on.
   - I self-reviewed my PR, ticking each file as "Viewed" once I was sure the changes were correct.  I encourage the reviewers to do the same, and to take all the time needed to make sure the changes make sense.  There's no rush.
   - I'm pushing this change at the beginning of the sprint, so that we have 2 weeks before the code freeze, and another 2 weeks in testing before this is released.
   - I went through all the changes and compiled a list of things to test.  I encourage both PR reviewers and beta testers to use the list I compiled above to ensure there are no regressions.
   - I'm pinging @startuptester to keep you in the loop.

3. What automated tests I added (or what prevented me from doing so)

This is a cleanup PR so we're just removing code that is not being used.  The reason why this cleanup is important is because keeping our codebase simple ensures we have something that's easy for us to maintain.

## PR submission checklist:
